datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -430,6 +445,11 @@ private static void configureDatasetResolvers(final RuntimeWiring.Builder builde
      * Configures resolvers responsible for resolving the {@link com.linkedin.datahub.graphql.generated.CorpUser} type.
      */
     private static void configureCorpUserResolvers(final RuntimeWiring.Builder builder) {
+        builder.type("CorpUser", typeWiring -> typeWiring
+            .dataFetcher("relationships", new AuthenticatedResolver<>(
+                new EntityRelationshipsResultResolver(GmsClientFactory.getRelationshipsClient())
+            ))
+        );
```

Comment:
if we have the ```.type("Entity", typeWiring -> typeWiring
                .dataFetcher("relationships", ...``` 

wiring above, why do we also need this one?

Reply:
this should not be needed!! mistake